src/js/models/appUser.js

```diff
@@ -42,7 +42,7 @@ var AppUser = module.exports = BaseModel.extend({
             return Backbone.Model.prototype.save.apply(this, _.toArray(arguments)).then(function() {
                 this._lastPropertyValues = this.pick(AppUser.EDITABLE_PROPERTIES);
                 return arguments;
-            });
+            }.bind(this));
```

Comment:
I hate it too, but that's a small thing to add support for those browsers. And `_(function(){}).bind(this)` looks almost the same ;)